<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>719</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>733</width>
    <height>719</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>733</width>
    <height>719</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <widget class="QListWidget" name="listAvailableDevices">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>500</width>
         <height>150</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>150</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 17pt &quot;Sitka&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
       </property>
       <property name="flow">
        <enum>QListView::Flow::TopToBottom</enum>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btnOnOffBluetooth">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>10</y>
         <width>85</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>OFF</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="listAvailableServices">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>220</y>
         <width>500</width>
         <height>150</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>150</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 17pt &quot;Sitka&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="flow">
        <enum>QListView::Flow::TopToBottom</enum>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>60</y>
         <width>152</width>
         <height>126</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 170, 127);
font: 800 18pt &quot;Sitka&quot;;</string>
          </property>
          <property name="text">
           <string>DEVICES</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnConnect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 85, 127);
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>SCAN</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>161</width>
         <height>123</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>159</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>159</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 170, 127);
font: 800 18pt &quot;Sitka&quot;;</string>
          </property>
          <property name="text">
           <string>SERVICES</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>156</width>
            <height>37</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QPushButton" name="btnSelectService">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 85, 127);
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>SELECT</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="listAvailableChar">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>380</y>
         <width>500</width>
         <height>150</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>150</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 17pt &quot;Sitka&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="flow">
        <enum>QListView::Flow::TopToBottom</enum>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>550</y>
         <width>666</width>
         <height>82</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnInput">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>83</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 85, 127);
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>INPUT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textWriteData">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 8pt &quot;Sitka&quot;;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 170, 127);
font: 800 18pt &quot;Sitka&quot;;</string>
          </property>
          <property name="text">
           <string>OUTPUT</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelReadData">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>161</width>
         <height>123</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 170, 127);
font: 800 18pt &quot;Sitka&quot;;</string>
          </property>
          <property name="text">
           <string>CHAR</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>156</width>
            <height>37</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QPushButton" name="btnSelectChar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 85, 127);
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>SELECT</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
